db.js

```diff
@@ -516,7 +516,9 @@ function upgrade (db) {
     store.createIndex(field + ', published', [field, PUBLISHED_FIELD], options)
   }
 
-  db.createObjectStore('settings', { keyPath: 'key' })
+  if (!db.objectStoreNames.contains('settings')) {
```

Comment:
Didn't fix, clearing browser cache is must. How to deal with this? Increment db version?

![Screenshot 2024-03-29 at 4 02 43 AM](https://github.com/hyphacoop/reader.distributed.press/assets/68826419/a7c0c839-da03-4c98-8a8e-14a3b6465ce7)